packages/perspective-node/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "perspective-node",
```

Comment:
Given the project is very simple and has a single package; I'd rather not use learna. But I don't know a lot about it; so feel free to educative me a little too if you think it's adding value here.